substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -385,15 +389,40 @@ impl CreateCmd {
 	}
 }
 
-/// Processes `CreateCmd` and returns string represenataion of JSON version of `ChainSpec`.
+/// Parses chain properties passed as a comma-separated KEY=VALUE pairs.
+fn parse_properties(raw: String) -> sc_chain_spec::Properties {
+	let mut props = sc_chain_spec::Properties::new();
+	for pair in raw.split(',') {
+		let mut iter = pair.splitn(2, '=');
+		let key = iter.next().unwrap_or("").trim().to_owned();
+		let value_str = iter.next().unwrap_or("").trim();
```

Comment:
If they don't exist, it should be an error.

Reply:
Fixed in [1b8e84a](https://github.com/paritytech/polkadot-sdk/pull/7368/commits/1b8e84a492dd696e0ca9cb9b29296bd5b8695758).